funtofem/interface/tacs_interface.py

```diff
@@ -977,7 +1030,7 @@ def create_from_bdf(
         thermal_index=-1,
         override_rotx=False,
         debug=False,
-        add_loads=False,
+        add_loads=True,  # whether it will try to add loads or not
```

Comment:
Good to add the comment, but why are we changing the default value?

Reply:
It checks whether the loads are there or not when it adds them.